schemas.py

```diff
@@ -0,0 +1,42 @@
+from pydantic import BaseModel, Field
+
+
+class UserIn(BaseModel):
+    first_name: str = Field(max_length=40)
+    last_name: str = Field(max_length=80)
+    email: str = Field(max_length=120)
+    password: str = Field(min_length=6)
+
+
+class User(BaseModel):
+    id: int
+    first_name: str = Field(max_length=40)
+    last_name: str = Field(max_length=80)
+    email: str = Field(max_length=120)
+    password: str = Field(min_length=6)
```

Comment:
Модели очень похожи, а значит можно создать одну из них, наследуясь от другой.
```suggestion
class User(UserIn):
    id: int
```